components/sensor/ade7953.rst

```diff
@@ -85,39 +77,54 @@ Additionally, the device has an ::doc:`NTC temperature sensor <ntc>`.
 
     sensor:
       - platform: ade7953
-        irq_pin: GPIO16
+        irq_pin: GPIO16 # Prevent overheating by setting this
+        update_interval: 10s
         voltage:
-          name: Shelly Voltage
-        current_a:
-          name: Shelly Current B
+          name: "${devicename} voltage"
```

Comment:
You're using substitutions here that aren't defined in this code block. For simplicity I'd make it just use Shelly, users who want substitutions can put them in themselves.